assets/dev/js/editor/controls/repeater.js

```diff
@@ -143,6 +153,7 @@ ControlRepeaterItemView = ControlBaseDataView.extend( {
 		}
 
 		this.toggleMinRowsClass();
+		this.maybeHideAddMoreButtonAndCopyButton();
```

Comment:
Not critical... these two methods are always called together, so you can create a new function like:

```
toggleClasses() {
    this.toggleMinRowsClass();
    this.toggleMaxRowsClass();
}
```

and just call `toggleClasses()` everywhere.

Reply:
Done